k8s/chart/values.yaml

```diff
@@ -0,0 +1,11 @@
+# Default values for chart.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+replicas: 1
+image:
+  registry: andrewstuart
```

Comment:
Got it, thanks @andrewstuart! :+1: 
Fine for now, in the future it could be nice if there was an official `uportal` or `apereo` prefix.